index.html

```diff
@@ -0,0 +1,175 @@
+
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css" />
+    <link rel="stylesheet" href="style.css">
+
+
+    <title>판다마켓</title>
+</head>
+<body>
+    <div class="conatiner">
+        <div class="header-wrapper">
+            <header>
+                <a class="logo" href="index.html">
+                    <div class="logo-image">
+                        <img src="image/판다 얼굴.png">
+                    </div>
+                    <div class="logo-text">
+                        <img src="image/판다마켓.png">
+                    </div>
+                </a>
+                <div class="login">
+                    <a class="login-button" href="login.html">
+                        <button class="login-button">
+                            로그인
+                        </button>
+                    </a>    
+                </div>
+            </header>
+        </div>
+
+        <div class="main-wrapper">
+            <main>
+                <section class="top-banner">
+                    <div class="top-banner">
+                        <div class="top-banner-text">
+                            <p>
+                                일상의 모든 물건을<br>
+                                거래해 보세요
+                            </p>
+                            <a class="find-items-button" href="items.html">
+                                <button class="find-items-button">
+                                    구경하러 가기
+                                </button>
+                            </a>
```

Comment:
❗️ 수정요청  
로그인 버튼을 눌렀을 때 'items.html'로 이동시켜주는 역할을 하니, 앵커 태그만 남겨두셔도 될 것 같습니다~  
버튼 요소처럼 보여도 꼭 버튼 태그로 작성하실 필요는 없습니다!  
해당 태그의 역할에 따라 태그를 작성하시는 것이 더 시맨틱하게 작성하는 방법입니다.

```suggestion
                            <a class="find-items-button" href="items.html"> 구경하러 가기</a>
```